build.gradle

```diff
@@ -80,6 +84,8 @@ configurations.all {
         // force testng dependency so we don't pick up a different version via GenomicsDB
         force 'org.testng:testng:' + testNGVersion
         force 'org.broadinstitute:barclay:' + barclayVersion
+        // make sure we don't pick up a different version via Picard
+        force googleCloudNioDependency
```

Comment:
I just realized this isn't sufficient. We actually need to forcibly reject the google component because it has a different name.